lib/home/profile/account/close_account.dart

```diff
@@ -29,12 +30,23 @@ class _CloseAccountState extends State<CloseAccount> {
           style: StyleText(size: 16),
         ),
         TextField(
-          onChanged: (value) {
-            final correctValue = value == 'DELETE';
+          obscureText: _obscureText,
+          onChanged: (value) async {
+            final correctValue = await context.read<_Deleting>().checkPassword(value);
```

Comment:
Nice work using `context.read()` in the callback!

(I think it's kinda confusing how you're supposed to use `.watch()` while the frame is being built and `.read()` when you're "between frames")